modules/v4.0.0/azure-storage-blob/security.tf

```diff
@@ -63,6 +63,8 @@ resource "azurerm_key_vault_secret" "storage_sas_secret" {
 }
 
 resource "azurerm_security_center_storage_defender" "datahub_storageaccount_defender" {
+  count = var.enable_defender > 0 ? 1 : 0
```

Comment:
Switching to a boolean, this would have to change slightly to something like this 

count = var.enable_defender ? 1 : 0